File: Status_19_07_2022/pattern1.c
#include <stdio.h>

void pattern(int row)
{
    int i = 1, j = 0;
    while (i <= row)
    {
        j = i;
        while (j--)
        {
            printf("*");
        }
        printf("\n");
            i++;
    }
}

int main()
{
    int num = 0;
    printf("Enter the number of rows:\n");
    scanf("%d", &num);
    pattern(num);
}

File: Status_19_07_2022/pattern1.py
#!/usr/bin/python3

from re import I


def pattern(row):
    i,j = 1,0
    while i <= row:
        j = i 
        while j != 0:
            print("*",end = "")
            j -=1
        print()
        i += 1
    
    
pattern(int(input("Enter the number of rows: ")))
        